src/plugins/currentLineMenu.js

```diff
@@ -131,7 +131,7 @@ function currentParagraphDecorations(doc, currentParagraph, editor) {
  */
 function decorationForCurrentParagraph(currentParagraph, editor) {
 	return Decoration.widget(
-		currentParagraph.pos + 1,
+		currentParagraph?.pos + 1,
```

Comment:
This will still cause an exception if currentParagraph is undefined, right? I'd expect `undefined + 1` to raise a type error. I'll make sure this never gets called without a paragraph instead.